discord/player.py

```diff
@@ -210,7 +210,12 @@ def _pipe_writer(self, source: io.BufferedIOBase) -> None:
             # arbitrarily large read size
             data = source.read(8192)
             if not data:
-                self._process.terminate()
+                if self._stdin is not None:
+                    self._stdin.close()
+                try:
+                    self._process.wait(timeout=60.0)
+                except subprocess.TimeoutExpired:
+                    self._process.terminate()
```

Comment:
```suggestion
```

Let's just do this instead since the process is cleaned outside of this function.

Reply:
Of course, that sounds good.
And sorry, I left a comment to your suggestion in the commit message by mistake.